src/main/java/org/sagebionetworks/web/client/PortalGinModule.java

```diff
@@ -1243,8 +1241,7 @@ protected void configure() {
 		bind(TableQueryResultWikiWidgetView.class).to(TableQueryResultWikiWidgetViewImpl.class);
 
 		bind(SingleButtonView.class).to(SingleButtonViewImpl.class);
-		bind(UserListView.class).to(UserListViewImpl.class);

```

Comment:
Deleted old UserListView (was only used by the challenge participant list widget)